README.md

```diff
@@ -1,23 +1,28 @@
 ![nsxiv](https://raw.githubusercontent.com/nsxiv/nsxiv/gh-pages/img/logo.png "nsxiv")
 
 **Neo Simple X Image Viewer**
+-----------------------------
 
-The sole purpose of nsxiv is to be the perfect image viewer for me. It is free
-software so that you can use it and modify it for your needs. Please file a bug
-report if something does not work as documented or expected. Contributions are
-welcome but there is no guarantee that they will be incorporated.
+nsxiv is a fork of sxiv with the purpose of trying to make sxiv
+suitable for general use, and to maintain it as sxiv is no more updated.
+It is free software so that you can use it and modify it for your needs.
```

Comment:
I think this can be worded a bit better. Although I don't have any suggestions atm.